### ISSUE TITLE
Fetch from remote fails.

### ISSUE THREAD

**Issue by Nageswari-droid** [neel1996/gitconvex#98]
```
2020/11/16 14:04:53 INFO: Fetching changes from ->  : refs/heads/:refs/remotes/
runtime error: invalid memory address or nil pointer dereference

goroutine 194 [running]:
runtime/debug.Stack(0x1, 0x0, 0x0)
        c:/go/src/runtime/debug/stack.go:24 +0xa5
runtime/debug.PrintStack()
        c:/go/src/runtime/debug/stack.go:16 +0x29
github.com/99designs/gqlgen/graphql.DefaultRecover(0x14f3c00, 0xc004020390, 0x1339b00, 0x18ba420, 0x0, 0x0)
        C:/Users/SVNPC/go/pkg/mod/github.com/99designs/gqlgen@v0.13.0/graphql/recovery.go:16 +0xae
github.com/99designs/gqlgen/graphql.(*OperationContext).Recover(0xc00015c500, 0x14f3c00, 0xc004020390, 0x1339b00, 0x18ba420, 0x8000, 0x2c18b271500)
        C:/Users/SVNPC/go/pkg/mod/github.com/99designs/gqlgen@v0.13.0/graphql/context_operation.go:110 +0x5d
github.com/neel1996/gitconvex-server/graph/generated.(*executionContext)._Mutation_fetchFromRemote.func1(0xc0047780c0, 0xc0047780f0, 0xc008a234e8)
        C:/Users/SVNPC/Desktop/js/gitconvex/graph/generated/generated.go:2888 +0x85
panic(0x1339b00, 0x18ba420)
        c:/go/src/runtime/panic.go:969 +0x1c7
github.com/go-git/go-git/v5/plumbing/transport/ssh.(*PublicKeysCallback).ClientConfig(0x0, 0xc004020900, 0x0, 0x2c1b27b5e38)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/plumbing/transport/ssh/auth_method.go:232 +0x2d
github.com/go-git/go-git/v5/plumbing/transport/ssh.(*command).connect(0xc002a701e0, 0x14ea7c0, 0x0)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/plumbing/transport/ssh/common.go:113 +0x58
github.com/go-git/go-git/v5/plumbing/transport/ssh.(*runner).Command(0xc000006610, 0x140b48d, 0xf, 0xc007bf23c0, 0x14ea7c0, 0x0, 0x2, 0x3, 0x0, 0xc0001339e0)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/plumbing/transport/ssh/common.go:52 +0x96
github.com/go-git/go-git/v5/plumbing/transport/internal/common.(*client).newSession(0xc0001785b0, 0x140b48d, 0xf, 0xc007bf23c0, 0x14ea7c0, 0x0, 0x0, 0x0, 0xc007bf23c0)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/plumbing/transport/internal/common/common.go:112 +0x90
github.com/go-git/go-git/v5/plumbing/transport/internal/common.(*client).NewUploadPackSession(0xc0001785b0, 0xc007bf23c0, 0x14ea7c0, 0x0, 0xc007bf23c0, 0x0, 0x0, 0x2c1b27aaa10)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/plumbing/transport/internal/common/common.go:89 +0x69
github.com/go-git/go-git/v5.newUploadPackSession(0xc00002ed50, 0x25, 0x14ea7c0, 0x0, 0x0, 0x6, 0x304df76f01, 0xc004772580)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/remote.go:378 +0xaa
github.com/go-git/go-git/v5.(*Remote).fetch(0xc004772740, 0x14f3b80, 0xc00002a0c0, 0xc008a231d8, 0x0, 0x0, 0x0, 0x0)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/remote.go:312 +0x150
github.com/go-git/go-git/v5.(*Remote).FetchContext(...)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/remote.go:286
github.com/go-git/go-git/v5.(*Repository).FetchContext(0xc0040205a0, 0x14f3b80, 0xc00002a0c0, 0xc008a231d8, 0xc0040207b0, 0xc002a0a280)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/repository.go:1114 +0xc5
github.com/go-git/go-git/v5.(*Repository).Fetch(...)
        C:/Users/SVNPC/go/pkg/mod/github.com/go-git/go-git/v5@v5.2.0/repository.go:1092
github.com/neel1996/gitconvex-server/git.FetchFromRemote(0xc0040205a0, 0x0, 0x0, 0x0, 0x0, 0xb9e8df)
        C:/Users/SVNPC/Desktop/js/gitconvex/git/git_fetch.go:39 +0x32b
github.com/neel1996/gitconvex-server/graph.(*mutationResolver).FetchFromRemote(0xc00008c040, 0x14f3c00, 0xc004020390, 0xc0086a85ae, 0x8, 0xc004778140, 0xc004778160, 0x20, 0x20, 0x2c18b271500)
        C:/Users/SVNPC/Desktop/js/gitconvex/graph/schema.resolvers.go:59 +0x11c
github.com/neel1996/gitconvex-server/graph/generated.(*executionContext)._Mutation_fetchFromRemote.func2(0x14f3c00, 0xc004020390, 0x20, 0x1399c00, 0xc004778101, 0xc004772140)
        C:/Users/SVNPC/Desktop/js/gitconvex/graph/generated/generated.go:2910 +0x1f6
github.com/99designs/gqlgen/graphql/executor.processExtensions.func3(0x14f3c00, 0xc004020390, 0xc004772140, 0xc0040203c0, 0xc0040203f0, 0x0, 0x0)
        C:/Users/SVNPC/go/pkg/mod/github.com/99designs/gqlgen@v0.13.0/graphql/executor/extensions.go:62 +0x42
github.com/neel1996/gitconvex-server/graph/generated.(*executionContext)._Mutation_fetchFromRemote(0xc0047780c0, 0x14f3c00, 0xc004020360, 0xc008926300, 0xc0047720c0, 0x2, 0x2, 0x0, 0x0)
        C:/Users/SVNPC/Desktop/js/gitconvex/graph/generated/generated.go:2908 +0x2c5
github.com/neel1996/gitconvex-server/graph/generated.(*executionContext)._Mutation(0xc0047780c0, 0x14f3c00, 0xc004020360, 0xc0089fc280, 0x1, 0x1, 0xc004020330, 0x30)
        C:/Users/SVNPC/Desktop/js/gitconvex/graph/generated/generated.go:6214 +0xff8
github.com/neel1996/gitconvex-server/graph/generated.(*executableSchema).Exec.func2(0x14f3c00, 0xc004020330, 0x10)
        C:/Users/SVNPC/Desktop/js/gitconvex/graph/generated/generated.go:943 +0x85
github.com/99designs/gqlgen/graphql/executor.(*Executor).DispatchOperation.func1.1.1(0x14f3c00, 0xc004020330, 0xc0047780d0)
        C:/Users/SVNPC/go/pkg/mod/github.com/99designs/gqlgen@v0.13.0/graphql/executor/executor.go:105 +0x4a
github.com/99designs/gqlgen/graphql/executor.processExtensions.func2(0x14f3c00, 0xc004020330, 0xc0047780d0, 0x14d2e20)
        C:/Users/SVNPC/go/pkg/mod/github.com/99designs/gqlgen@v0.13.0/graphql/executor/extensions.go:59 +0x42
github.com/99designs/gqlgen/graphql/executor.(*Executor).DispatchOperation.func1.1(0x14f3c00, 0xc004020240, 0xc004020210)
        C:/Users/SVNPC/go/pkg/mod/github.com/99designs/gqlgen@v0.13.0/graphql/executor/executor.go:104 +0x131
github.com/99designs/gqlgen/graphql/handler/transport.POST.Do(0x14f1dc0, 0xc000008000, 0xc00015c400, 0x14f0300, 0xc00015ebd0)
        C:/Users/SVNPC/go/pkg/mod/github.com/99designs/gqlgen@v0.13.0/graphql/handler/transport/http_post.go:53 +0x3c8
github.com/99designs/gqlgen/graphql/handler.(*Server).ServeHTTP(0xc000229be0, 0x14f1dc0, 0xc000008000, 0xc00015c400)
        C:/Users/SVNPC/go/pkg/mod/github.com/99designs/gqlgen@v0.13.0/graphql/handler/server.go:115 +0x21b
github.com/gorilla/mux.(*Router).ServeHTTP(0xc0002c6600, 0x14f1dc0, 0xc000008000, 0xc00015c100)
        C:/Users/SVNPC/go/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0xe5
github.com/rs/cors.(*Cors).Handler.func1(0x14f1dc0, 0xc000008000, 0xc00015c100)
        C:/Users/SVNPC/go/pkg/mod/github.com/rs/cors@v1.7.0/cors.go:219 +0x1c6
net/http.HandlerFunc.ServeHTTP(0xc000229d60, 0x14f1dc0, 0xc000008000, 0xc00015c100)
        c:/go/src/net/http/server.go:2042 +0x4b
net/http.serverHandler.ServeHTTP(0xc00016a1c0, 0x14f1dc0, 0xc000008000, 0xc00015c100)
        c:/go/src/net/http/server.go:2843 +0xaa
net/http.(*conn).serve(0xc004ea80a0, 0x14f3b40, 0xc0029d4000)
        c:/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
        c:/go/src/net/http/server.go:2969 +0x36d

```

**Comment by Nageswari-droid** [comment]
```
2020/11/18 10:38:21 ERROR: Authentication method failed -> error creating SSH agent: "SSH agent requested but Pageant not running"

```

**Comment by neel1996** [comment]
@Nageswari-droid fix pushed as part of 73c4ea8de1efc033de20210ced60e73da7b2ac9f
